src/examples/source/FIAMS_FullScan_Unknown_test.cpp

```diff
@@ -45,49 +45,56 @@ void test_main_FIAMS_FullScan_Unknown()
 
   std::cout << "mt1 size: " << mt1.getSmallMoleculeSectionRows().size() << std::endl;
   std::cout << "mt2 size: " << mt2.getSmallMoleculeSectionRows().size() << std::endl;
-  assert(mt1.getSmallMoleculeSectionRows().size() == 7865);
+  assert(mt1.getSmallMoleculeSectionRows().size() == 7669);
 
-  const OpenMS::Feature* f1 = &fm1[0];
-  const OpenMS::Feature* f2 = &fm2[0];
-  std::cout << "0 peak_apex_int: " << f1->getMetaValue("peak_apex_int") << std::endl;
-  std::cout << "0 getRT: " << f1->getRT() << std::endl;
-  const auto* a1 = &mt1.getSmallMoleculeSectionRows().front();
-  const auto* a2 = &mt2.getSmallMoleculeSectionRows().front();
-  std::cout << "0 calc_mass_to_charge: " << a1->calc_mass_to_charge.get() << std::endl;
-  std::cout << "0 chemical_formula: " << a1->chemical_formula.get() << std::endl;
-  std::cout << "0 identifier: " << a1->identifier.get().at(0).get() << std::endl;
-  assert(Utilities::assert_close((double)f1->getMetaValue("peak_apex_int"), 3.5241834375e05));
-  assert(Utilities::assert_close((double)f1->getRT(), 0.0));
-  assert(Utilities::assert_close((double)a1->calc_mass_to_charge.get(), 102.957));
-  assert(static_cast<std::string>(a1->chemical_formula.get()) == "H3O3P");
-  assert(a1->identifier.get().at(0).get() == "HMDB:HMDB0001443");
-  assert(Utilities::assert_close((double)f1->getIntensity(), (double)f2->getIntensity()));
-  assert(Utilities::assert_close((double)f1->getMetaValue("peak_apex_int"), (double)f2->getMetaValue("peak_apex_int")));
-  assert(Utilities::assert_close((double)f1->getRT(), (double)f2->getRT()));
-  assert(Utilities::assert_close((double)a1->calc_mass_to_charge.get(), (double)a2->calc_mass_to_charge.get()));
-  assert(static_cast<std::string>(a1->chemical_formula.get()) == static_cast<std::string>(a2->chemical_formula.get()));
-  assert(a1->identifier.get().at(0).get() == a2->identifier.get().at(0).get());
+  // First feature
+  {
+    const auto& f1 = fm1[0].getSubordinates()[0];
+    const auto& f2 = fm2[0].getSubordinates()[0];
```

Comment:
`getSubordinates()` is required to read correct values of meta data such as `peak_apex_int`.
I was inspired by this: https://github.com/OpenMS/OpenMS/blob/d9692da0d410c06b6cdc960f608a5c962360d09c/src/tests/class_tests/openms/source/MRMFeatureSelector_test.cpp#L97